arches_lingo/serializers.py

```diff
@@ -62,33 +30,31 @@ def repair_right_statement(self, instance):
         return instance
 
 
-class SchemeLabelSerializer(ArchesModelSerializer):
+# Generic serializers for Lingo.
+class LingoResourceSerializer(ArchesModelSerializer):
     class Meta:
         model = ResourceInstance
-        graph_slug = "scheme"
-        nodegroups = ["appellative_status"]
+        graph_slug = None  # generic
+        nodegroups = "__all__"
         fields = "__all__"
+        read_only_graphs = ["group", "person", "textual_work"]
 
 
-class SchemeLabelTileSerializer(ArchesTileSerializer):
+class LingoTileSerializer(ArchesTileSerializer):
     class Meta:
         model = TileModel
-        graph_slug = "scheme"
-        root_node = "appellative_status"
+        graph_slug = None  # generic
+        root_node = None  # generic
         fields = "__all__"
 
-    def validate(self, data):
-        data = super().validate(data)
+    def validate_appellative_status(self, data):
```

Comment:
The next thing we're going to want is a similar hook in the resource serializer to ensure that if you're using that, you can run this logic too. I'm still thinking that part through, but at least there's no functionality regression with this because Johnathan's work on labels uses the tile serializer.